Index: app/src/main/java/com/example/utocka/HttpClient.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/utocka/HttpClient.kt b/app/src/main/java/com/example/utocka/HttpClient.kt
--- a/app/src/main/java/com/example/utocka/HttpClient.kt	
+++ b/app/src/main/java/com/example/utocka/HttpClient.kt	(date 1701634767746)
@@ -13,9 +13,7 @@
 
 fun SendRolesUpdate(username:String, role:String){
     val client = OkHttpClient()
-
     val url = "http://localhost/api/user-role"
-
     val json = """
         {
             "username": "$username",
